src/views/Home/components/PrizeTracks/PrizeTracks.scss

```diff
@@ -0,0 +1,67 @@
+@use "~stylesheets/typography";
+
+.prizeTracks {
+  background: linear-gradient(#b0b6f7, #b0b6f7);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  &__title {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    &__container {
+      width: 524px;
+      max-width: 90%;
+      height: 106px;
+      background-color: #32578E;
+      border-radius: 28px;
+      display: flex;
+      justify-content: center;
+      align-items: center; 
+      &--header {
+        @include typography.title;
+        font-size: 70px;
+        text-shadow: 5px 0px #777CE7;
+      }
+    }
+  }
+  
+  &__images {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    align-items: center;
+    &__container {
+      margin: 40px 50px;
+      width: 341px;
+      min-width: 260px;
+      aspect-ratio: 1/1;
+      border-radius: 50%;
+      background-color: rgba(255, 255, 255, 0.6);
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-shadow: -4px 8px 4px rgba(20, 28, 94, 0.4); 
+      &--hover {
+        background-color: #162C7B;
+        opacity: 0.76;
+        color: white;
+        border-radius: 50%;
+        width: inherit;
+        aspect-ratio: 1/1;
+        position: absolute;
+        z-index: 100;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        @include typography.title;
+        font-style: italic;
+        text-align: center;
```

Comment:
<img width="384" alt="tracks-smaller-phones" src="https://user-images.githubusercontent.com/38510322/147324529-78b693c3-6f67-4013-9bd1-fb26e88fcb30.PNG">

On smaller screens, I've noticed when you click the image to show the circle outline and the size is exceeding length of the screen.

Reply:
Can I ask how you're displaying the content on your phone? I tried using ngrok but it seems like auth0 doesn't like that

Reply:
I've seen some people have issues with auth0 on mobile devices still. This is my setup but I'm not sure if this will work for everyone. 

I use a windows machine + iPhone, on my windows machine I'm connected to home Wi-Fi and I have it set as a private network in my laptop settings. On my phone, I am connected to the same Wi-Fi and I use the network address given from React console after the server has been setup on my phone. On non-authenticated routes, it seems to work well for me but I know some people have had issues with auth0. If you want, you can comment out the Auth0 Provider within `src/auth/Auth0ProviderWithHistory.tsx` and try non-authenticated routes on your phone. 

In the end, we still run it on the development branch to catch these issues since I know it's difficult to test for everyone.

Reply:
Sounds good, I think I fixed it but I only have an android so I cant test for apple rendering